src/pages/pupils/lessons/[lessonSlug]/results/[attemptId].tsx

```diff
@@ -0,0 +1,11 @@
+import withFeatureFlag from "@/hocs/withFeatureFlag";
+
+const CanonicalResultsPage = () => {
+  return <div>CanonicalResultsPage</div>;
+};
+
+const FeaturedCanonicalResultsPage = withFeatureFlag(
```

Comment:
Actually we don't need the flag once we get to the page.  So just export CanonicalResultsPage as default and remove the rest